src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -0,0 +1,208 @@
+/**
+ * ProjectData.java - this file implements the Project class, which contains details about a
+ * particular project, including the members and tasks within it.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+
+enum User {
```

Comment:
I would suggest renaming this to clarify the difference between this and the UserData object.  From the looks of it, something like "UserProjectRole" might make sense